.gitignore

```diff
@@ -8,3 +8,5 @@ build
 tmp
 node_modules/
 .env
+VERSION
```

Comment:
why do not commit the file?

Reply:
This file is automatically generated when building the charms, based on the most recent git tag. We don't need to keep track of it.
If we version this file, it will complicate development and merging pull requests, as it will certainly conflict.